docs/celo-codebase/protocol/transactions/index.md

```diff
@@ -1,6 +1,5 @@
 ---
-title: Transactions
-id: transactions
+title: Overview
```

Comment:
There is a page titled "Overview" for every section. What do you think about renaming the title to "Transactions Overview" for this file, and prefixing the section title to "Overview" for every overview page?

Reply:
I can do that no problem